src/main.rs

```diff
@@ -691,6 +689,12 @@ impl From<ignore::Error> for ProgramError {
     }
 }
 
+impl From<tree_sitter::QueryError> for ProgramError {
```

Comment:
Similar, can use `TSQueryError` (main motivation: "See all references" on `TSQueryError` then finds everything)

Reply:
Fixed in https://github.com/alexpovel/srgn/pull/144/commits/5c08ce08de67f30c77d691e4c0309efbe31f2bec